xml/ftp_banners.xml

```diff
@@ -1287,4 +1287,17 @@ more text</example>
     <param pos="2" name="service.version"/>
     <param pos="1" name="host.name"/>
   </fingerprint>
+  <fingerprint pattern="^(?:(\S+) )?FTP server \(QVT\/Net ([\d.]+)\) ready\.?$">
+    <description>QVT/Net FTP Server</description>
+    <example host.name="siren" service.version="5.1">siren FTP server (QVT/Net 5.1) ready.</example>
+    <example host.name="qpc-qvtnet" service.version="4.1">qpc-qvtnet FTP server (QVT/Net 4.1) ready.</example>
+    <param pos="0" name="os.vendor" value="Microsoft"/>
+    <param pos="0" name="os.device" value="General"/>
```

Comment:
Removed @tsellers-r7 - there was a reference in the Vermillion FTPD fingerprint I added last week too